docroot/sites/settings/cgov_settings_utilities.inc

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * @file
+ * Cgov utility functions for use throughout the *.settings.php files
+ *
+ * Make sure to use require_once() so this file is never loaded more than
+ * once per page.
+ */
+
+
+/**
+ * Returns the current environment name for use in determining settings.
+ *
+ * @return string
+ *   The environments name
+ */
+function settings_get_environment() {
+  if (file_exists('/var/www/site-php') && isset($_ENV['AH_SITE_ENVIRONMENT'])) {
+    $env = $_ENV['AH_SITE_ENVIRONMENT'];
+    if (preg_match('/^ode\d*$/', $env)) {
+      $env = 'ode';
+    }
+  } else {
+    $env = 'local';
+  }
+  return $env;
+}
+
+/**
+ * Returns the current domain name for use settings.
+ *
+ * @return string
+ *   The primary domain name
+ */
+function settings_get_domain() {
+  // Several of our settings files require the current domain.
+  // Default our domain to PROD.
+  $domain = 'www.cancer.gov';
+  // Check if this is an ACSF site first.
+  if ($is_acsf_env && $acsf_db_name) {
+    $domains = gardens_data_get_sites_from_file($GLOBALS['gardens_site_settings']['conf']['acsf_db_name']);
+    // You should always add the public domain first domain when creating a site,
+    // but just in case we will set to the default first.
+    $domain = array_keys($domains)[0];
+    foreach ($domains as $site_name => $site_info) {
+      if (!empty($site_info['flags']['preferred_domain'])) {
+        $domain = $site_name;
+      }
+    }
+  // Then check if this is a different Acquia Cloud site.
+  } else {
+    $env = settings_get_environment();
+    // On local we just want to use the default.
```

Comment:
Do we want to use the default (prod URL) or `www.devbox` on local? Nowhere else in our code do we hardcode `www.devbox`. It's only in the docker and Apache configuration.

Reply:
This is something @bryanpizzillo and I talked about a bit.  What it came down to is that the only url we're going to support for local development is `www.devbox`.